python/sdk/test/transformer/sim_exp_resp_feast_w_tracing.json

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
Curious - how was this baseline file generated? When I use the internal /simulate endpoint in the dev env, I see some differences in the operation tracing:

<img width="1183" alt="Screenshot 2023-10-10 at 9 47 58 AM" src="https://github.com/caraml-dev/merlin/assets/23465343/493563b0-7ea6-4048-bdd7-1ca4dd288491">

<img width="1174" alt="Screenshot 2023-10-10 at 9 47 40 AM" src="https://github.com/caraml-dev/merlin/assets/23465343/e5a9925b-6c75-458c-af61-900a76ed4cfb">

<img width="1336" alt="Screenshot 2023-10-10 at 9 47 23 AM" src="https://github.com/caraml-dev/merlin/assets/23465343/71ecf7cc-dd00-4370-a985-ae65ba0c0633">

Could you double-check this?

For things like "Source=Bigtable", we can perhaps ignore comparing them as it will depend on the deploy-time config. Wdyt?